ui/components/app/currency-input/currency-input.js

```diff
@@ -44,13 +44,14 @@ export default function CurrencyInput({
 
   const [isSwapped, setSwapped] = useState(false);
   const [newHexValue, setNewHexValue] = useState(hexValue);
+  const [shouldDisplayFiat, setShouldDisplayFiat] = useState(featureSecondary);
 
   const shouldUseFiat = () => {
```

Comment:
I know this function existed before now but does `shouldUseFiat` need to be a function?  Can it just be a ternary?

Reply:
@darkwing Changed `shouldUseFiat` function to be a ternary [here](https://github.com/MetaMask/metamask-extension/pull/13827/commits/0d2eba062b090bef6d230583d0568efe8885ce83).